projects/addhub.md

```diff
@@ -0,0 +1,25 @@
+Addhub
+--------------------
+
+### Background:
+We are going to set a user-defined advetisement push/pull platform.
+
+The basic function of the platform:
+user log into the platform, post his/her advertisement with keywords, such as:
+"I would like to sell a sofa.I am Brooklyn. The sofa price is $50."
+"keywords: sofa, brooklyn, $50"
+Then s/he choose which public platforms s/he'd like to push, such as craiglist and facebook seconf forum and local buzzsell forum.
+The pisted advertisement will be pushed to multiple web platforms.
```

Comment:
Minor, but maybe use 1., 2., 3., etc...

Instead of `s/he` write `they`. 

Craiglist is mispelled, it's Craigslist.

What's "facebook seconf" forum?

Typo: `posted` -> `posted`.